cmd/argocd/commands/project.go

```diff
@@ -18,8 +18,31 @@ import (
 	"github.com/argoproj/argo-cd/pkg/apis/application/v1alpha1"
 	"github.com/argoproj/argo-cd/server/project"
 	"github.com/argoproj/argo-cd/util"
+	"github.com/spf13/pflag"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type projectOpts struct {
+	description  string
+	destinations []string
+}
+
+func (opts *projectOpts) GetDestinations() []v1alpha1.ApplicationDestination {
+	destinations := make([]v1alpha1.ApplicationDestination, 0)
+	for _, destStr := range opts.destinations {
+		parts := strings.Split(destStr, ",")
+		if len(parts) != 2 {
+			log.Fatalf("Expected destination of the form: server;namespace. Received: %s", destStr)
```

Comment:
Wondering if `server;namespace` (with a semicolon) might have been meant to be `server,namespace`.

Reply:
Nice catch! fixed